app/CocoaPods.xcodeproj/project.pbxproj

```diff
@@ -543,6 +543,7 @@
 			baseConfigurationReference = C511D44E7F9B7A0C0EEF51D0 /* Pods.release.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CODE_SIGN_IDENTITY = "Developer ID Application";
```

Comment:
It's pretty much the same, though I'm happy to move it over, there's not _too_ much [DI](https://github.com/CocoaPods/CocoaPods-app/blob/master/Rakefile#L7) work going on in the rakefile.